tools/make-updatedhelp.ps1

```diff
@@ -0,0 +1,141 @@
+<#
+    Usage:
+
+    1. Clone the docs respository
+       cd C:\temp
+       git clone https://github.com/MicrosoftDocs/PowerShell-Docs.git
+
+    2. Run make-updatedhelp.ps1 for the version folder you want to build
+
+      .\make-updatedhelp.ps1 -sourceFolder C:\temp\PowerShell-Docs\reference\5.1 -Verbose
+
+    3. Run Update-Help  to install the newly built help
+
+       Update-Help -SourcePath c:\temp\updatablehelp\5.1 -Recurse -Force
+#>
+
+
+param(
+    [Parameter(Mandatory=$true)]
+    [ValidateScript({ Test-Path $_ })]
+    [string]$sourceFolder
+)
+
+$tempDir = [System.IO.Path]::GetTempPath()
+
+# Pandoc source URL
+$panDocVersion = "2.7.3"
+$pandocSourceURL = "https://github.com/jgm/pandoc/releases/download/$panDocVersion/pandoc-$panDocVersion-windows-x86_64.zip"
+
+$docToolsPath = New-Item (Join-Path $tempDir "doctools") -ItemType Directory -Force
+$pandocZipPath = Join-Path $docToolsPath "pandoc-$panDocVersion-windows-x86_64.zip"
+Write-Verbose "Downloading Pandoc..."
+Invoke-WebRequest -Uri $pandocSourceURL -OutFile $pandocZipPath
+
+Expand-Archive -Path $pandocZipPath -DestinationPath $docToolsPath -Force
+$pandocExePath = Join-Path $docToolsPath "pandoc-$panDocVersion-windows-x86_64\pandoc.exe"
+
+$platyPSversion = "0.14.0"
+Write-Verbose "Downloading platyPS..."
+Save-Module -Name platyPS -Repository PSGallery -Force -Path $docToolsPath -RequiredVersion $platyPSversion
+Import-Module -FullyQualifiedName $docToolsPath\platyPS\$platyPSversion\platyPS.psd1
+
+$DocSet = Get-Item $sourceFolder
+$WorkingDirectory = $PWD
+
+function Get-ContentWithoutHeader {
+    param(
+        $path
+    )
+
+    $doc = Get-Content $path -Encoding UTF8
+    $start = $end = -1
+
+    # search the first 30 lines for the Yaml header
+    # no yaml header in our docset will ever be that long
+
+    for ($x = 0; $x -lt 30; $x++) {
+        if ($doc[$x] -eq '---') {
+            if ($start -eq -1) {
+                $start = $x
+            }
+            else {
+                if ($end -eq -1) {
+                    $end = $x + 1
+                    break
+                }
+            }
+        }
+    }
+    if ($end -gt $start) {
+        Write-Output ($doc[$end..$($doc.count)] -join "`r`n")
```

Comment:
Would be better to use `[Environment]::Newline` unless the tooling only works with Windows style newlines

Reply:
The intent of this tool is Windows-only but I will change it. It could be targeted at any docs for any platform.